### DISCUSSION TITLE
Problem: configure layernames according to "Data Specification on Elevation" - Elevation model View service

### DISCUSSION THREAD

**Discussion by sebros123** [INSPIRE-MIF/helpdesk#89] [Q&A] [answered]
Hi, 

We encounter a problem when we try to follow the layernamestandard in the data specification, 
we do not think that the layers we have in our service are correct agree with the layernames defined in the data specification.

Our service contains these layers: 
"Terränglutning" = Terrain slope
![image](https://user-images.githubusercontent.com/92439786/156574235-53d9da1a-573c-4ea9-8225-7cce3bcf78d5.png)

"Terrängskuggning" = hill shading
![image](https://user-images.githubusercontent.com/92439786/156574125-a0171f76-d8c6-4621-a02d-cc5303cb3991.png)

How can we best match with these layersnames?

![image](https://user-images.githubusercontent.com/92439786/156574860-bed58c82-1c63-41e8-ab3a-f72f5d03f84f.png)

best regards, Sebastian

**Comment by sMorrone** [comment, marked as answer]
Dear @sebros123
in case the data has been harmonised, you have to use the name of related spatial data object type. However, keep in mind that according to [endorsed change in Req. 39 of the View Service TG ](https://github.com/INSPIRE-MIF/technical-guidelines/issues/2) "The layer name can be “as is”, in case the data served are not harmonised according to [INS DS]". The idea being discussed and already implemented for testing in the staging instance of the Validator - see [issue #39](https://github.com/INSPIRE-MIF/helpdesk-validator/issues/39) - is that a harmonised View service can serve both as-is and harmonised data, only the latter being obliged to follow the IR naming rules. The Validator will then check for the presence of at least one layer with conforming name.
Hope this helps

**Comment by sMorrone** [comment]
I am moving this issue to the "Discussions" tab and classify it as "[Q&A](https://github.com/INSPIRE-MIF/helpdesk/discussions/categories/q-a)"
